var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -22,8 +22,8 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
 
     version("develop", branch="develop")
     version("master", branch="master")
-    version("glu", branch="glu")
-    version("glu_experimental", branch="glu_experimental")
+    version("1.5.0.glu", branch="glu")
+    version("1.5.0.glu_experimental", branch="glu_experimental")
```

Comment:
```suggestion
    version("1.5.0.glu_experimental", branch="glu_experimental")
```
I suggest removing all references to Ginkgo branch `glu` as this branch is about to be deleted. The plan is to merge branch `glu_experimental` into main Ginkgo development branch.

Reply:
That's a good point, and might need to be propagated back to https://github.com/spack/spack/pull/32499, or will this be closed?